### PR TITLE
javascript exercise

### PR DESCRIPTION
**Pull request by juanazadian**
Simple web implemented for the javascript exercise. 
The style was done with bootstrap but the modal style was customized separately.

### CODE REVIEW

File: js/ex4js.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        <h1>New Item</h1>
+        <form id="form" action=#>
+          <label for="name">Name:</label><br>
+          <input type="text" id="name" name="name"><br>
+          <label for="cost">Cost:</label><br>
+          <input type="text" id="cost" name="cost"><br><br>
+          <input type="submit" id="create" value="Create">
+        </form>
+      </div>
+
```
Comment (jpbalarini) [inline js/ex4js.html]: remove empty new line

File: js/style.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+@import "bootstrap-5.0.2/scss/bootstrap.scss";
+
+.modal-juan {
```
Comment (jpbalarini) [inline js/style.scss]: haha, `juan` looks like a non descriptive class. Let's use something like `custom-modal` or something similar

File: js/scripts.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  cell.innerHTML = nameValue;
+  cell = row.insertCell();
+  cell.innerHTML = costValue;
+  cell = row.insertCell();
+  cell.innerHTML = '<input type="checkbox" name="done1" value="done1">';
+  modal.style.display = "none";
+  event.preventDefault();
+}
+
+optim.onclick = function(){
```
Comment (jpbalarini) [inline js/scripts.js]: let's improve this. You could use an Array sort (with a custom sort function) to generate a sorted list and then iterate that list  and use the `appendChild`  function.

**Review by jpbalarini — CHANGES_REQUESTED** [review]
Left some comments. 
Also, let's add boostrap 5 to the root of the project structure so we don't have to import it in each part (you could remove boostrap 4 from the previous parts)

**Comment by juanazadian** [comment]
image

**Pull request merged**